src/app/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,39 @@
+import defaultAvatar from './defaultAvatar.svg';
+import classes from './Profile.module.css';
+import { useCurrentUser } from '../../utils/user';
+
+function openLoginDialog() {
+  overwolf.profile.openLoginDialog();
+}
+
+const User = (): JSX.Element => {
```

Comment:
This component looks pretty good, clean and understandable now. With all the logic outsourced. 🏆

Reply:
thanks